test/unit-tests/broker-connection.test.js

```diff
@@ -0,0 +1,26 @@
+const expect = require('chai').expect;
+const io = require('socket.io-client');
+
+describe('Broker connection handling', () => {
```

Comment:
It is not evident from the test what we're testing here. 🤔

Reply:
Changed it!